aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -1343,6 +1372,9 @@ def main(args=None, *, prog="pg_migrate"):
     else:
         logging.basicConfig(level=logging.INFO, format=log_format)
 
+    if not args.filtered_roles:
+        args.filtered_roles = ()
```

Comment:
the parser has already the support for default values. but this can't work. should be an empty string as default value, I think.

then, down there, you could do something like

```filtered_roles=[ role for role in args.filtered_roles.split(",") if role != ]``` (untested code)

so that the filtered_roles sequence is actually empty when passing it to the main func.